chapters/03_queueing_model/05_numeric_results_and_timings/main.tex

```diff
@@ -0,0 +1,331 @@
+    \includegraphics[width=\textwidth]{chapters/03_queueing_model/img/numeric_results_and_timings/truncation_effect/waiting/demo.pdf}
+    \caption{
+        Comparison of mean waiting time between values obtained from the Markov 
+        chain formula, values obtained from the truncated simulation and values
+        obtained from the untruncated simulation.
+    }
+    \label{fig:markov_vs_des_waiting_time_comparison_overall}
+\end{figure}
+
+In detail, Figure~\ref{fig:markov_vs_des_waiting_time_comparison_overall} shows
+the calculated mean waiting time using the Markov chain, using a truncated 
+simulation and using a simulation with infinite capacity (without the artificial
+parameters \(N\) and \(M\)).
+Each plot corresponds to different values of \(N\) and \(M\) and is run over 
+different values of \(\lambda_2\).
+The untruncated simulation values are the same at all three graphs since
+the effect of truncation does not apply to it.
+The waiting times generated by the truncated simulation match the ones generated 
+by the Markov chains model.
+Note that this comparison includes both type 1 and type 2 individuals.
+% TODO: Include type 1 and type 2 plot?
+
+
+Similarly Figure~\ref{fig:markov_vs_des_blocking_time_comparison_overall} shows
+the mean blocking time equivalent comparison between the three approaches used
+for the waiting time (Markov chain, truncated simulation and untruncated
+simulation).
+Similar to the waiting time, the blocking time among the different approaches
+begin to get closer together as the value of \(N\) and \(M\) increases.
+Note that the blocking time can only be calculated for type 2 individuals.
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth]{chapters/03_queueing_model/img/numeric_results_and_timings/truncation_effect/blocking/demo.pdf}
+    \caption{
+        Comparison of mean waiting time between values obtained from the Markov 
+        chain formula, values obtained from the truncated simulation and values
+        obtained from the untruncated simulation.
+    }
+    \label{fig:markov_vs_des_blocking_time_comparison_overall}
+\end{figure}
+
+Finally, Figure~\ref{fig:markov_vs_des_proportion_within_time_comparison_overall}
+shows the overall proportion of individuals whose time in the system are within
+a time target for different values of \(N\) and \(M\).
+Similar to the previous figures, as \(N\) and \(M\) increase the proportion of
+individuals between the simulation and the Markov chain get closer.
+
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth]{chapters/03_queueing_model/img/numeric_results_and_timings/truncation_effect/proportion/demo.pdf}
+    \caption{
+        Comparison of mean waiting time between values obtained from the Markov 
+        chain formula, values obtained from the truncated simulation and values
+        obtained from the untruncated simulation.
+    }
+    \label{fig:markov_vs_des_proportion_within_time_comparison_overall}
+\end{figure}
+
```

Comment:
Can you add a paragraph/sentence here concluding with values used throughout the rest of the work?

Reply:
I don't fully understand. What do you mean by values? Parameter values?

Reply:
Add a sentence to reflect on the choice of the artificial values for the truncation effect